.gitignore

```diff
@@ -1,5 +1,7 @@
 *.iml
+*.json
 .gradle
+/.idea
```

Comment:
We shouldn't ignore all the idea folder, see [https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems)